### ISSUE TITLE
Incorret `forge:tools/axes` tag entry

### ISSUE THREAD

**Issue by PssbleTrngle** [Laokuai233/Twilight-Delight#26]
There is an entry in the `forge:tools/axes` tag file that does not exists, thus throwing an error an deleting the entire item tag.

```java
[03Sep2022 10:20:13.939] [Worker-Main-8/ERROR] [net.minecraft.tags.TagLoader/]: Couldn't load tag forge:tools/axes as it is missing following references: minecraft:fiery_axe (from TwilightDelight-1.18.2-1.2.0-Alpha.jar), 
	minecraft:ironwood_axe (from TwilightDelight-1.18.2-1.2.0-Alpha.jar), 
	minecraft:knightmetal_axe (from TwilightDelight-1.18.2-1.2.0-Alpha.jar), 
	minecraft:steeleaf_axe (from TwilightDelight-1.18.2-1.2.0-Alpha.jar)
```

Not sure why it's happening, I looked in the code and the content of  the tag file there mentions `twtilightforest:ironwood_axe`, but the tag file in the released mod jar has `minecraft:ironwood_axe`

**Comment by Goulixiaoji** [comment]
Yes, we have found the problem before and fixed it in 1.2.0-Beta. Perhaps you can update the mod.

**Comment by PssbleTrngle** [comment]
I found the beta in the github releases, but is there a reason it is not uploaded to curseforge?

**Comment by PssbleTrngle** [comment]
I still get an error with the beta:

`Couldn't load tag forge:tools/axes as it is missing following references: twilightforest:fiery_axe`

There is no fiery axe, but you have included it in your tag.
Same for the fiery shovel

**Comment by Aceplante** [comment]
Couldn't load tag forge:tools/axes as it is missing following references: minecraft:fiery_axe (from TwilightDelight-1.18.2-1.1.4.jar), 
	minecraft:ironwood_axe (from TwilightDelight-1.18.2-1.1.4.jar), 
	minecraft:knightmetal_axe (from TwilightDelight-1.18.2-1.1.4.jar), 
	minecraft:steeleaf_axe (from TwilightDelight-1.18.2-1.1.4.jar)
